force-app/main/default/lwc/communityConfirmation/communityConfirmation.html

```diff
@@ -8,7 +8,9 @@
             </span>
         </div>
         <div class="slds-col slds-size_10-of-12">
-            <h1 class="navds-heading navds-heading--large spacing-bottom spacing-top">{title}</h1>
+            <h1 class="navds-heading navds-heading--large spacing-bottom spacing-top" role="alert" aria-live="polite">
+                {title}
+            </h1>
         </div>
```

Comment:
`<div role="alert" class="sr-only">
            {title}
        </div>
        <div class="slds-col slds-size_10-of-12">
            <h1 class="navds-heading navds-heading--large spacing-bottom spacing-top">{title}</h1>
        </div>`

Reply:
Og css:
```
.sr-only {
  position: absolute;
  width: 1px;
  height: 1px;
  padding: 0;
  margin: -1px;
  overflow: hidden;
  clip: rect(0, 0, 0, 0);
  white-space: nowrap;
  border: 0;
}
```